molecule/vault/molecule.yml

```diff
@@ -0,0 +1,42 @@
+---
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  # docker + systemd recipe courtesy https://github.com/pilou-
+  - name: vault-host
+    image: xenial
+    command: /sbin/init
+    security_opts: ['seccomp=unconfined']
```

Comment:
Note that [Fedora seccomp profile](https://src.fedoraproject.org/rpms/docker/raw/master/f/seccomp.json) could be used.

Reply:
I should really learn more about apparmor. It is also needed in the context of codejail, a library used by Open edX which requires specific apparmor settings to be tested.

Reply:
In which context was this Fedora seccomp created and why is it a good fit in this case?

Reply:
This Fedora seccomp profile whitelists system calls required by Systemd.